stage1_fly/stage1_fly.mk

```diff
@@ -9,6 +9,8 @@ $(call setup-stamp-file,FLY_STAMP,aci-build)
 
 $(call inc-many,$(foreach sd,$(FLY_SUBDIRS),$(sd)/$(sd).mk))
 
+$(call inc-one,../stage1/stop/stop_fly.mk)
```

Comment:
It would probably be cleaner if the stop command for the fly stage1 image had a separate source code. It would even be a bit simpler because in fly stage1 image, there are no children processes at all.